### PR TITLE
Handle staffinfo

### PR DESCRIPTION
**Pull request by nick-benoit14**
* Adds support for 'STAFFINFO' .dat files

### CODE REVIEW

File: lib/blacklight/xml_parser.rb [hunk 1]
```diff
@@ -11,6 +11,22 @@ module Blacklight
     course: "Course",
     questestinterop: "Assessment",
     content: "Content",
+    staffinfo: "StaffInfo",
+
+    # categories: :iterate_categories,
```
Comment (bfcoder) [inline lib/blacklight/xml_parser.rb]: Why add these back in? Do we still need them?

File: lib/blacklight/models/staff_info.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        </ul>
+      HTML
+    end
+
+    def canvas_conversion(course)
+      page = CanvasCc::CanvasCC::Models::Page.new
+      page.body = construct_body
+      page.identifier = @id
+
+      page.page_name = if @title.empty?
```
Comment (bfcoder) [inline lib/blacklight/models/staff_info.rb]: With so little code maybe consider one line?
```ruby
page.page_name = @title.empty? ? "Contact" : @title
```

**Review by srphillips — APPROVED** [review]
I'm srphillips and I approve this merge request.

**Review by bfcoder — APPROVED** [review]
Looks good!

File: lib/blacklight/models/staff_info.rb [hunk 3]
```diff
@@ -0,0 +1,71 @@
+
+module Blacklight
+  class StaffInfo
+    attr_accessor(
```
Comment (kamijean) [inline lib/blacklight/models/staff_info.rb]: Jamis was telling me that unless you need to edit the values on the class from another file then have this be attr_reader. In looking at the code, I don't know if you are accessing very many of these except in your specs. And then to access private variables you can do something like (@question.instance_variable_get :@max_score).

**Pull request merged**